SwiftLint/swiftlint.sh

```diff
@@ -1,10 +1,31 @@
+#!/bin/bash
 if [ -z "$CI" ]; then
 	set -e
 	
     if which swiftlint >/dev/null; then
     	BASEDIR=$(dirname "$0")
-        swiftlint --config "$BASEDIR/.swiftlint-source.yml"
-        swiftlint --config "$BASEDIR/.swiftlint-tests.yml" || true # Don't fail if there's no tests to lint
+    	count=0
+
+	    for file_path in $(git ls-files -m --exclude-from=.gitignore | grep ".swift$"); do
+	        export SCRIPT_INPUT_FILE_$count=$file_path
+	        count=$((count + 1))
+	    done
+
+	    for file_path in $(git diff --name-only --cached | grep ".swift$"); do
```

Comment:
@Boris-Em I would love your feedback on those two for loops. Are there any other git commands we need to take into account? E.g. when files are staged, already committed or pushed, etc. I feel like you've got more knowledge in this matter!